atlasdb-conjure/src/test/java/com/palantir/atlasdb/http/v2/DialogueClientOptionsTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.http.v2;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.nio.file.Paths;
+import java.util.List;
+
+import org.junit.Test;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.palantir.atlasdb.config.AuxiliaryRemotingParameters;
+import com.palantir.atlasdb.config.ImmutableServerListConfig;
+import com.palantir.atlasdb.config.ServerListConfig;
+import com.palantir.conjure.java.api.config.service.PartialServiceConfiguration;
+import com.palantir.conjure.java.api.config.service.ServicesConfigBlock;
+import com.palantir.conjure.java.api.config.service.UserAgent;
+import com.palantir.conjure.java.api.config.ssl.SslConfiguration;
+
+public class DialogueClientOptionsTest {
+    private static final String SERVICE_NAME = "service";
+    private static final UserAgent USER_AGENT = UserAgent.of(UserAgent.Agent.of(SERVICE_NAME, "1.2.3"));
+    private static final List<String> SERVERS = ImmutableList.of("apple", "banana", "cherry", "dewberry");
+    private static final SslConfiguration SSL_CONFIGURATION = SslConfiguration.of(Paths.get("a", "b"));
+
+    private static final AuxiliaryRemotingParameters REMOTING_PARAMETERS_EXTENDED_TIMEOUT
+            = AuxiliaryRemotingParameters.builder()
+            .userAgent(USER_AGENT)
+            .shouldRetry(true)
+            .shouldLimitPayload(true)
+            .shouldUseExtendedTimeout(true)
+            .build();
+    private static final AuxiliaryRemotingParameters REMOTING_PARAMETERS = AuxiliaryRemotingParameters.builder()
```

Comment:
name it _SHORT

Reply:
Yep, makes sense - this sounds like a default.